### PR TITLE
Show GE and HA only when showing both prices

### PR DESCRIPTION
**Pull request by deathbeam**
- Change the price displaying to dropdown menu
- Show (GE:, (HA: only when showing both prices at once, otherwise do
not add label at all

Depends on: #2149

Preview:
screenie

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java [hunk 1]
```diff
@@ -402,7 +402,7 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 				}
 			}
 
-			if (config.showMenuItemQuantities() && itemComposition.isStackable() && quantity > 1)
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: Why is this removed?

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: Because now entire right click menu can be disabled from being recolored, so I felt like having option for this is unnecessary as it is good default behaviour.
